### ISSUE TITLE
D2VSource rejects LF line endings

### ISSUE THREAD

**Issue by qyot27** [Asd-g/MPEG2DecPlus#6]
I was attempting to use a .d2v file generated by [D2VWitch](https://github.com/dubhater/D2VWitch) but when using a simple `D2VSource("test.d2v")` script, it kept failing with:
```
[ffmpeg/demuxer] avisynth: MPEG2Source: Could not open one of the input files.
[ffmpeg/demuxer] (testver.avs, line 1)
```
It was fine with a .d2v created with DGIndex, but that's not too helpful for Linux (or macOS, etc.) users that would prefer a fully-native solution instead of needing to resort to Wine to get the .d2v created.

It *seems* that D2VSource expects the .d2v file to have Windows CRLF (\r\n) line endings, and D2VWitch outputs just LF (\n).  If I `todos` the output from D2VWitch, it works.

**Comment by Asd-g** [comment]
I cannot reproduce the issue on Windows. Using the file generated by D2VWitch (LF line endings) works fine.
I don't have a Linux at the moment but will install and will check.

**Comment by qyot27** [comment]
As I didn't mention it before, tests were done on Ubuntu 20.10.  It probably would also show up under WSL, so having a separate Linux VM or partition isn't necessarily required.

A little bit more testing with an available test file:
https://samples.ffmpeg.org/MPEG2/interlaced/

Using `koncercik.[mpg|d2v]`, I could still see the line ending issue, but a bigger issue is the file path part: if it was an absolute path (the default), it fails no matter if it's CRLF or LF.  If I use the relative path (either by editing it down myself in a text editor or using d2vwitch's relative paths switch), then the LF endings fail, but CRLF works.

And from even further testing, it's very specifically that the CRLF needs to be at the end of the file path line.  I deliberately changed just that line to CRLF in a file with only LF, and it started working.  But doing the same with the absolute path still failed.

So to summarize, just as it pertains to line 3,

|  | File.mpg\r\n | File.mpg\n
| - | ------------- | ------------
Absolute path | Fails | Fails
Relative path | Works | Fails

And it would have to be isolated to the non-WIN32 condition in the if/else branch handling the file path:
https://github.com/Asd-g/MPEG2DecPlus/commit/a2451b0d9c81567a0aa28ef69b03a509382b29eb#diff-a31a65341c799df0aedc2d1a68db3fdfe19df23fd4e767c9091f26421f294a52R251

For what it's worth, it can only parse DGIndex's output if it uses relative paths too.  If you use DGIndex's 'Use Full Paths' option, the result is `Z:\home\user\Videos\koncercik.mpg` or similar.  But parsing it down to just the filename works.

**Comment by Asd-g** [comment]
Please try again with the latest change.

**Comment by qyot27** [comment]
Yep, that appears to have fixed it.
